local/bin/py/build/actions/integrations.py

```diff
@@ -1017,4 +1017,9 @@ def process_manifest(self, manifest_json, name):
             manifest_json["name"] = manifest_json.get("name", name)
             if skipped_tags:
                 print(f'\x1b[33mWARNING\x1b[0m: Categories skipped on integration {manifest_json["name"]}, {skipped_tags}')
+
+        # temporary workaround until we can source integration data from APW
+        # https://datadoghq.atlassian.net/browse/WEB-4579
+        if 'oauth' in manifest_json.keys(): del manifest_json['oauth']
```

Comment:
Also just thinking you might be able to use pop() with a default to remove oauth without the key check